plugins/python-build/share/python-build/pyston-0.6.1

```diff
@@ -0,0 +1,23 @@
+case "$(pyston_architecture 2>/dev/null || true)" in
+"linux64" )
+  install_package "pyston-0.6.1-linux64" "https://github.com/dropbox/pyston/releases/download/v0.6.1/pyston-0.6.1-linux64.tar.gz#b09516d8e421cb004ef39d832bae600face4ecf80d9791469fb2c026bcf1faae" "pyston" verify_py27
```

Comment:
It seems pyston executable is linked against so many libraries.

```
% ldd pyston
        linux-vdso.so.1 (0x00007ffe583f7000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fdb6eaa8000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007fdb6e88e000)
        libreadline.so.6 => /lib/x86_64-linux-gnu/libreadline.so.6 (0x00007fdb6e644000)
        libsqlite3.so.0 => /usr/lib/x86_64-linux-gnu/libsqlite3.so.0 (0x00007fdb6e33c000)
        libgmp.so.10 => /usr/lib/x86_64-linux-gnu/libgmp.so.10 (0x00007fdb6e0b9000)
        libmpfr.so.4 => /usr/lib/x86_64-linux-gnu/libmpfr.so.4 (0x00007fdb6de50000)
        libssl.so.1.0.0 => not found
        libcrypto.so.1.0.0 => not found
        libutil.so.1 => /lib/x86_64-linux-gnu/libutil.so.1 (0x00007fdb6dc4d000)
        liblzma.so.5 => /lib/x86_64-linux-gnu/liblzma.so.5 (0x00007fdb6da27000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fdb6d81f000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fdb6d619000)
        libtinfo.so.5 => /lib/x86_64-linux-gnu/libtinfo.so.5 (0x00007fdb6d3ef000)
        libatomic.so.1 => /usr/lib/x86_64-linux-gnu/libatomic.so.1 (0x00007fdb6d1e7000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fdb6cee3000)
        libstdc++.so.6 => /usr/lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fdb6cb61000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fdb6c94a000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fdb6c5aa000)
        /lib64/ld-linux-x86-64.so.2 (0x000055748da7a000)
```

Actually the pyston executable won't run even on Ubuntu 16.04 if some of runtime dependencies are missing. Let's just show some warning about binary compatibility issue. I don't think installing every dependencies isn't worth doing.